src/components/ActionBar/ActionBar.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import ActionBar from './ActionBar';
+import Mic from '../Icons/Mic';
+import Video from '../Icons/Video';
+
+const defaultRequiredProps = {
+  toggleMicrophone: () => {},
+  toggleCamera: () => {},
+  toggleScreenSharing: () => {},
+  endCall: () => {},
+};
+
+describe('<ActionBar /> component', () => {
+  it('should render without crashing', () => {
+    const component = shallow(<ActionBar {...defaultRequiredProps} />);
+
+    expect(component).toHaveLength(1);
+  });
+
+  it('snould render <Mic /> component when microphone turned on', () => {
```

Comment:
typo `snould` instead of `should`

Reply:
[Done](https://github.com/interviewr/raven-client/pull/39/commits/8382cdc194cf77de251cb00c60e8a6abdfca45aa#diff-9e645d2f387bae8dd6a2d3cfad28d8e4R21)